localisation/00_wc_random_artifact_names.csv

```diff
@@ -177,7 +177,7 @@ artifact_jewelry_rare_prefix_intrigue_2;Disgraced's;;Des Entehrten;;;;;;;;;;;x
 artifact_jewelry_rare_prefix_intrigue_3;Enveloped;;Umh�llter;;;;;;;;;;;x
 artifact_jewelry_rare_prefix_intrigue_4;Snake's;;Finsterer;;;;;;;;;;;x
 artifact_jewelry_rare_prefix_intrigue_5;Spider's;;Der Spinne;;;;;;;;;;;x
-artifact_jewelry_rare_prefix_intrigue_6;Malevolence;;B�swilliger;;;;;;;;;;;x
+artifact_jewelry_rare_prefix_intrigue_6;Malevolent;;B�swilliger;;;;;;;;;;;x
```

Comment:
@arithon, check localization changes, pls

Reply:
only these two lines? 180 and 400? looks good, I think

Reply:
Yeah, only these two lines. Thanks <3